app/routines/process_spreadsheet.rb

```diff
@@ -3,25 +3,34 @@ class ProcessSpreadsheet
 
   protected
 
-  def exec(filename:, offset:, pad_cells: true, &block)
+  def exec(filename:, offset: 1, pad_cells: true, headers: false, &block)
```

Comment:
Maybe a comment here to make it easier to tell this isn't to include or disable them, but a symbol to call on them?